application/features/bootstrap/EmailContext.php

```diff
@@ -1168,7 +1168,7 @@ public function thatUserHasAPasswordThatExpiresInDays($n)
 
         $currentPassword = $this->tempUser->currentPassword;
         $currentPassword->password = base64_encode(random_bytes(33)); // Needed to pass validation
-        $currentPassword->expires_on = MySqlDateTime::relative('+' . $n . ' days');
+        $currentPassword->expires_on = MySqlDateTime::relative($n . ' days');
```

Comment:
This fixed an email test where it was being passed `-1` and strtotime started to be unable to handle `+-1days` in our php8.3, but it worked in our php8.1.

Reply:
tricky...good fix!